website/documentation/contribute/code/roles/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Community Roles
+weight: 20
+---
+
+
+## Community Roles
+
+This document describes contribution roles within the Gardener community, outlining their requirements and responsibilities. It aims to help new contributors understand the review process and provides guidance for those interested in gaining membership. Various aspects and definitions were adopted from the Kubernetes Community membership document ([ref](https://github.com/kubernetes/community/blob/master/community-membership.md)), with slight adjustments  to fit the organizational setup and size of Gardener.
+
+> [!Note]
+> Not every Gardener sub-project may have applied the listed community roles yet. 
+> If adopted, this document is linked in the repository's top-level `README.md` file.
+
+### Member
+
+Members are active contributors in the Gardener community. They can have issues and PRs assigned to them, and are granted the privilege to run pre-submit tests on their own and all other PRs.
+Technically, members are defined by their membership in the [Gardener Github org](https://github.com/gardener).
+
+**Requirements**
+- Demonstrate stake and long-term commitment in the Gardener project, e.g., through previous code contributions or company affiliation
+- Maintained Github profile to make yourself known to other project maintainers
+- Enabled Github notifications for their ongoing PR reviews and issues
+
+**Responsibilities**
+- Responsive to their assigned PRs, issues and mentions
+
+**How to become a member**
+- Fulfill listed requirements
+- Obtain sponsorship from at least two approvers
+- **Process**: Open an issue in the affected project repository (use [issue template](https://github.com/gardener/documentation/blob/master/website/documentation/contribute/code/roles/assets/_issue_template.md); if accepted, an org owner assigns the member to the corresponding group.
+
+### Reviewer
+
+Reviewers are responsible for assessing code quality and correctness within specific areas of the project.
+They are granted the privilege to `/lgtm` pull requests.
+Members of this role are technically defined in the `OWNERS_ALIASES` file, placed at the root level of the repository.
```

Comment:
As we have `Members` defined above, it might make sense to not use it in this context.

```suggestion
Contributors with this role are technically defined in the `OWNERS_ALIASES` file, placed at the root level of the repository.
```